packages/core/App.tsx

```diff
@@ -46,9 +49,16 @@ export default function App(props: AppProps) {
     const hasQuerySelected = useSelector(selection.selectors.hasQuerySelected);
     const isDarkTheme = useSelector(selection.selectors.getIsDarkTheme);
     const shouldDisplaySmallFont = useSelector(selection.selectors.getShouldDisplaySmallFont);
+    const isDisplayingSmallScreenWarning = useSelector(
+        interaction.selectors.getIsDisplayingSmallScreenWarning
+    );
+    const hasDismissedSmallScreenWarning = useSelector(
+        interaction.selectors.getHasDismissedSmallScreenWarning
+    );
     const platformDependentServices = useSelector(
         interaction.selectors.getPlatformDependentServices
     );
+    const [measuredNodeRef, , measuredWidth] = useLayoutMeasurements<HTMLDivElement>();
```

Comment:
Blank arg because having weird eslint issues here, open to suggestions

Reply:
I tried doing that, but it kept throwing the error :/ It's working elsewhere in the codebase though so I'm not sure what the exception is here